src/pages/guidelines/carbon-for-ai/index.mdx

```diff
@@ -0,0 +1,590 @@
+sub-titleColor="dark"
+hoverColor="dark"
+href="/components/number-input/usage/#ai-presence">
+
+![AI number input](images/mini-number-input.png)
+
+</ImageCard>
+
+<ImageCard
+sub-title="Text input"
+sub-titleColor="dark"
+hoverColor="dark"
+href="/components/text-input/usage/#ai-presence">
+
+![AI text input](images/mini-text-input.png)
+
+</ImageCard>
+
+</Column>
+
+<Column colMd={3} colLg={3} noGutterSm>
+
+<ImageCard
+sub-title="Dropdown"
+sub-titleColor="dark"
+hoverColor="dark"
+href="/components/dropdown/usage/#ai-presence">
+
+![AI dropown](images/mini-dropdown.png)
+
+</ImageCard>
+
+<ImageCard
+sub-title="Radio button"
+sub-titleColor="dark"
+hoverColor="dark"
+href="/components/radio-button/usage/#ai-presence">
+
+![AI radio button](images/mini-radio-button.png)
+
+</ImageCard>
+
+<ImageCard
+sub-title="Tile"
+sub-titleColor="dark"
+hoverColor="dark"
+href="/components/tile/usage/#ai-presence">
+
+![AI tile](images/mini-tile.png)
+
+</ImageCard>
+
+</Column>
+
+</Row>
+
+## Accessibility
+
+When designing an AI experience, ask does the user know AI is being used? The
+base principle of Carbon for AI is to mark where AI is present while providing
+explainability whenever available. This level of transparency helps build trust
+between users and the system. This transparency is crucial as the line between
+AI and human is becoming increasingly blurred.
+
+#### Tested for color contrast
+
+The Carbon for AI style elements and components are tested for accessibility to
+ensure that colors, type, and contrast adhere to Carbon’s existing accessibility
+standards.
+
+#### AI label provides accessible AI visibility
+
+The AI label is the primary indicator to communicate that AI is present. To
+ensure that the AI label remains accessible across light and dark themes, use
+the provided component.
+
+#### Light spread is limited for readability
+
+Carbon for AI utilizes light sources to distinguish it from base Carbon
+components. These light sources glow on container edges, but the spread is
+limited and subtle to ensure accessible contrast with UI content.
+
+<Row>
+<Column colLg={8}>
+
+<Tabs>
+
+<Tab label="Light mode">
+
+![Demonstration of text color contrast in a light mode.](images/ai-accessibility-light.png)
+
+</Tab>
+
+<Tab label="Dark mode">
+
+![Demonstration of text color contrast in a dark mode.](images/ai-accessibility-dark.png)
+
+</Tab>
+
+</Tabs>
+
+</Column>
+</Row>
+
+## AI resources
+
+<Row className="resource-card-group">
+
+<Column colLg={4} colMd={4} noGutterSm>
+  <ResourceCard
+    subTitle="IBM Design for AI"
+    href="https://w3.ibm.com/w3publisher/design-for-ai"
+    actionIcon="launch">
+    <MdxIcon name="bee" />
+  </ResourceCard>
+</Column>
+</Row>
+
+## Feedback
+
+Help us improve this component by providing feedback, asking questions, and
+leaving any other comments on GitHub. For IBMers, questions and feedback can be
+directed to Slack in the channels #carbon-design-system or #carbon-for-ai.
```

Comment:
Should we add links to slack?
[#carbon-design-system](https://ibm-studios.slack.com/messages/C0M053VPT/)
[#carbon-for-ai](https://ibm-studios.slack.com/messages/C0603LZUKRV)

Reply:
Oh nice those Slack links actually work. I feel like it used to not link very nicely to Slack.